readme.md

```diff
@@ -50,37 +49,65 @@ Adds filetype glyphs (icons) to other plugins such as [NERDTree], [vim-airline],
 <a name="quick-setup"></a>
 ## Quick Setup (TL;DR)
 
-1. Install the plugin per your usual method
-   _[(» More details... «)](#installation)_
-2. Download and install a patched font (or patch your own):
+1. Download and install a patched font (or patch your own):
```

Comment:
Fix broken link after nerd-fonts: "(» More details... «)" I suggest making it link to nerd-fonts also (a bit redundant which is okay and maybe just in case) or simply remove it and leave the "nerd-fonts" link :smiley: